### ISSUE TITLE
SourceLink information not retained after copy/pasting package to a different location.

### ISSUE THREAD

**Issue by TrymKristian** [dotnet/sourcelink#904]
Due to legal reason I cannot disclose any screenshots of this issue to the public.

I have a set of nuget packages that are being built on Azure Devops Server with SourceLink. These  nuget packages are dumped by a pipeline to a fileshare. So far so good.

However when I copy the nuget package onto my local machine. The SourceLink information just vanishes. 
I've observed this issue by leveraging the NuGet Package Explorer.  

Is this an intended feature with SourceLink. Or do I need to investigate potential firewalls between me and the fileShare.

**Comment by TrymKristian** [comment]
No, symbols are its own .snupkg.

**Comment by TrymKristian** [comment]
Yeah, we're using jfrog with the newly supported artifactory for .snupkg to serve as our symbol server. I was thinking of using source link. So using nupkg and snupkg I've been able to debug packages. However I just found this issue very strange that, when as mentioned, I just copied a nupkg from network location onto local disk.

**Comment by tmat** [comment]
@clairernovotny I'm guessing that somehow NuGet Package Explorer can't find the PDBs/symbol packages. 
Source Link information is embedded into the PDB. The tool that uses Source Link info needs to be able to find the PDB.
